### ISSUE TITLE
Be able to rearrange clips using keyboard navigation

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#42]
# What's the problem you're hoping this new feature will solve? 
If someone is not using a mouse, clips for a message cannot be re-ordered. That is a fairly common action, and it's a barrier to getting the full experience of the app.

# Description of feature
This feature would allow people using a keyboard or other non-mouse device to move clips up or down in the sort order of a message on both the message create page, and the post-create edit screen.

# Alternatives and workarounds
You have to use a mouse or trackpad to re-order clips.

# Additional context
Github has a decent implementation of this on their issue listing pages that reveals itself when tabbing through, but doesn't clutter up the mouse-driven reordering experience.

See the arrow in the middle (focused) item in this screenshot for an example:
![image](https://user-images.githubusercontent.com/342510/108902602-f09f5f80-75e1-11eb-9345-03249f315d5a.png)

**Comment by robby1066** [comment]
This [issue on the sortablejs repo](https://github.com/SortableJS/Sortable/issues/1954) appears to point at a possible starting point on this.

**Comment by robby1066** [comment]
See the [Sortable documentation](https://github.com/SortableJS/Sortable/blob/master/README.md#methods) for some starting points on how to accomplish this.

**Comment by robby1066** [comment]
![Screenshot 2021-03-15 at 18 25 14](https://user-images.githubusercontent.com/342510/111234203-08a04880-85bc-11eb-8b6e-d29012fe69b2.gif)

**Comment by robby1066** [comment]
[Here's a writeup of the thought process behind how this issue was resolved.](https://robbymacdonell.medium.com/refactoring-a-sortable-list-for-keyboard-accessibility-2176b34a07f4)

**Comment by robby1066** [comment]
This is implemented in development now. Needs to be deployed to production and then will do final testing and close the issue.

**Comment by robby1066** [comment]
Tested and deployed on March 18
